modules/virt-about-container-disks.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-using-containerdisk-volumes.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-container-disk-with-datavolumes.adoc
+
+[id="virt-about-container-disks_{context}"]
+= About container disks 
+
+A container disk is a virtual machine image stored as a container image in a container image registry. Container disks can deliver the same disk images to multiple virtual machines and can be used to create large numbers of virtual machine clones.
+
+A container disk can either be imported into a persistent volume claim (PVC) using a DataVolume and attached to a virtual machine, or attached directly to a virtual machine as an ephemeral `containerDisk` volume.
+
+== Importing a container disk into a PVC using a DataVolume
+
+The Containerized Data Importer (CDI) imports the container disk into a PVC using a DataVolume. The DataVolume can then be attached to a virtual machine for persistent storage.
+
+== Attaching a container disk to a virtual machine as a containerDisk volume
+
+A containerDisk volume is ephemeral. It is discarded when
```

Comment:
Correct me if I'm wrong, but anywhere you use `containerDisk` should be marked up in backticks, I think, according to the new guidelines.

Reply:
Yep, and all the more important here to help distinguish between container disks.